README.md

```diff
@@ -98,6 +99,14 @@ Regardless of platform, the following libraries/dependencies are required:
 5. Run `mklink openloco.exe bin\` or `copy openloco.exe bin\`
 6. Run the game, `bin\openloco`
 
+Building using cmake:
+Note: The game can currently only be built for 32-bit architectures
+ To install dependencies, run `.\vcpkg.exe install sdl2:x86-windows sdl2-mixer:x86-windows zlib:x86-windows libpng:x86-windows yaml-cpp:x86-windows breakpad:x86-windows`
```

Comment:
These will be rendered as one paragraph unless they're separated by another newline.